vars/maven.groovy

```diff
@@ -119,4 +120,15 @@ def mvnSetVersionProperty(MavenCommand mvnCmd, String property, String newVersio
         .withProperty('allowSnapshots', true)
         .withProperty('generateBackupPoms', false)
         .run('versions:set-property')
-}
\ No newline at end of file
+}
+
+def uploadLocalArtifacts(String mvnUploadCredsId, String artifactDir, String repoUrl) {
```

Comment:
I would replace this by
`def uploadLocalArtifacts(String mvnUploadCredsId, String artifactDir, String repoUrl, String zipFileName = 'artifacts')`
(and remove next line)
wdyt?

Reply:
do we really need to change the name of the `zipFileName` as it is all internal to the method ?